R/zzz.R

```diff
@@ -0,0 +1,19 @@
+gradethis_default_options <- list(
+  gradethis_glue_correct = "{ random_praise() } { .message } { .correct }",
+  gradethis_glue_incorrect = "{ .message } { .incorrect } { random_encourage() }",
+
+  # nolint start
+  gradethis_glue_pipe = "I see that you are using pipe operators (e.g. %>%), 
```

Comment:
use a `paste0` as after the new line, currently there are many spaces that will be included